notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    plot_victim_visit_sweep(df, "$\\texttt{Latest}$", "adv-vs-cp505-vary-visits")
</pre><p>It's great this is all already factored into a helper method!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>